components/03-modules/dashboard-table/_dashboard-table.scss

```diff
@@ -0,0 +1,21 @@
+$dashboard-table-margin: 30px !default;
+
+.dashboard-table {
+    &__header {
+        display: flex;
+        flex-wrap: nowrap;
+        align-items: flex-end;
+        margin-bottom: $dashboard-table-margin;
+        padding-bottom: 10px;
+        border-bottom: 1px solid $gray;
+    }
+
+    &__content {
+        margin-bottom: $dashboard-table-margin;
+    }
+
+    &__title {
+        margin-bottom: 0;
+        margin-right: $dashboard-table-margin;
+    }
+}
```

Comment:
We should make the table complex - in mobile view, the labels should be displayed inside each cell. Table component should be corrected, `data-label` visible only for mobile version should be added. Please add separate task for this and add this feature later for alpaca components.

Reply:
done, refactored `table` element and it's now showing with `data-th` 😄

Reply:
great, I've added dome comments to this, but it looks better 👍